internal/messages/broadcast.go

```diff
@@ -6,6 +6,10 @@ import "github.com/hashicorp/memberlist"
 // being converted into a Broadcast.
 //
 // onDone will be called once the message has been broadcasted or invalidated.
+//
+// Queueing the resulting Broadcast will invalidate all previous broadcasts
+// for messages with the same name; this means that callers should only queue
+// newer messages.
```

Comment:
Just curious; this isn't new functionality but documenting the existing one right?

Reply:
Right, this has always been true but I didn't understand that it was true before.